.env.example

```
@@ -1,2 +1,7 @@
 DATABASE_URL=postgresql://admin:secret@localhost:5432/mydb
-PORT=3000
\ No newline at end of file
+PORT=3000
+
+JWT_ACCESS_SECRET=your_access_secret_here
+JWT_ACCESS_EXPIRES_IN=15m    # 15 minutes
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_ACCESS_EXPIRES_IN key should go before the JWT_ACCESS_SECRET key

(UnorderedKey)